gem_rbs_collectionによる型情報のインストール

a5ob7r opened the pull request:
# 概要

利用しているサードパーティーのgemの型情報をインストールする。これにより、katakata_irbが利用できる型情報が増えるので、補完能力も強化される。

リポジトリのREADMEに従って導入する。とても簡単だ。

```sh
bundle exec rbs collection init
bundle exec rbs collection install
```

https://github.com/ruby/gem_rbs_collection

a5ob7r committed:
Add gem_rbs_collection

Install it as below. This improves completions with katakata_irb.

```sh
bundle exec rbs collection init
bundle exec rbs collection install
```

tk0miya commented on README.md:
````diff
@@ -10,6 +10,14 @@ docker compose run --rm app bin/setup
+Install type signatures.
+
+```sh
+docker compose exec app bundle exec rbs collection install
````
脱線:
まだ細かい運用のところは検討しきれていないのですが、各個人の環境では `--frozen` オプションつけた方がいいかもです。
単に `rbs collection install` すると、最新版の型情報を検索しにいき、 `rbs_collection.lock.yaml` が更新される事があるようです。
(`rbs collection update` とのすみ分けがあまりわかっていない)

a5ob7r replied:
npmのiとciみたいなものですかね。

tk0miya replied:
ですね。 `--frozen` オプションは .lock ファイルを参照してインストールするはず。
`bundle install` にも同オプションがあったはず。

a5ob7r replied:
これかー。

```ruby
        unless params[:frozen]
          Collection::Config.generate_lockfile(config_path: config_path, definition: Bundler.definition)
        end
        Collection::Installer.new(lockfile_path: lock_path, stdout: stdout).install_from_lockfile
```

https://github.com/ruby/rbs/blob/b32c335e00318996556d8a1d4b9019ceb2735378/lib/rbs/cli.rb#L1102C8-L1102C8

a5ob7r replied:
実態としては、--frozenオプションでロックファイルの生成をスキップしているんですね。

a5ob7r replied:
READMEのインストールコマンドに--frozenオプションを追加しました。

a5ob7r committed:
Add --frozen option to rbs collection installation

Without this option, the installation command probably regenerate the
lockfile even if it already exists.
